gossip/src/cluster_info_metrics.rs

```diff
@@ -237,6 +242,11 @@ pub(crate) fn submit_gossip_stats(
             stats.process_gossip_packets_time.clear(),
             i64
         ),
+        (
+            "process_gossip_packets_itrs_since_last_report",
+            stats.process_gossip_packets_itrs_since_last_report.clear(),
+            i64
+        ),
```

Comment:
for easier queries, can you please report this in the same table as the other ones?